src/components/vr_cooperation/src/vr_module.cc

```diff
@@ -240,17 +240,11 @@ void VRModule::SendMessageToHMI(const vr_hmi_api::ServiceMessage& msg) {
   if (!proxy_.Send(msg)) {
     LOG4CXX_ERROR(logger_, "Couldn't send GPB message");
   }
-  if(vr_hmi_api::RESPONSE == msg.rpc_type()) {
-    request_controller_.DeleteRequest(msg.correlation_id());
-  }
 }
 
 void VRModule::SendMessageToMobile(application_manager::MessagePtr msg) {
   LOG4CXX_DEBUG(logger_, "Message to mobile: " << msg->json_message());
   service()->SendMessageToMobile(msg);
-  if(application_manager::MessageType::kResponse == msg->type()) {
```

Comment:
when response is timeout request already deleted when OnTimeoutTriggered() of request_controller